gui-wm/hyprland/hyprland-0.29.0.ebuild

```diff
@@ -75,10 +95,16 @@ pkg_setup() {
 src_prepare() {
 	if use video_cards_nvidia; then
 		cd "${S}/subprojects/wlroots" || die
-		eapply "${S}/nix/wlroots-nvidia.patch"
+		eapply "${S}/nix/patches/wlroots-nvidia.patch"
+		# https://bugs.gentoo.org/911597
+		# https://github.com/hyprwm/Hyprland/pull/2874
```

Comment:
The pull request was merged so the sed line isnt needed anymore.

Reply:
Thanks, removed it